<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QOverlayedControl</class>
 <widget class="QWidget" name="QOverlayedControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#QOverlayControl {
background-color: transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="styleFrame">
     <property name="styleSheet">
      <string notr="true">QFrame#styleFrame {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #303060,
stop:0.3 #101020,
stop:1 #202040);
border-radius: 10px;
border: 2px solid #888;
};</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QMoveHandle" name="labMoveHandle">
          <property name="minimumSize">
           <size>
            <width>37</width>
            <height>10</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>37</width>
            <height>10</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../common.qrc">:/player/move-handle</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,0,0">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QClickAndGoSlider" name="posSlider_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
   background-color: #555;
   height:           10px;
   border-radius:    3px;
}

QSlider::handle:horizontal {
   background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #727272, stop:0.15 white, stop:0.75 white, stop:1 #888);
   border: 1px solid #5c5c5c;
   border-radius: 3px;
   width:         4px;
   margin:       -6px 0;
 }

QSlider::handle:horizontal:disabled {
   border: 0;
   width:  0;
   margin: 0;
 }

QSlider::handle:horizontal:pressed, QSlider::handle:horizontal:hover {
   background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #002255, stop:0.15 #8ec6e9, stop:0.75 #8ec6e9, stop:1 #003366);
   border-color:     #003366;
}

QSlider::sub-page:horizontal, QSlider::sub-page:horizontal:disabled {
   background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #bacbdb, stop:0.280 #5077ae, stop:0.281 #406da6, stop:0.660 #b9ddf3, stop:0.950 #88bbe8, stop:0.951 #84b4da, stop:1 #84b4da);
   border: 1px solid #1e4785;
   border-radius:    3px;
}
</string>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="pageStep">
           <number>0</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>2</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QTimeLabel" name="labPos_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#labPos_2 {
color: #aaa;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string> 0:00:00</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushPlay">
          <property name="maximumSize">
           <size>
            <width>34</width>
            <height>34</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Play selected Channel</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../common.qrc">
            <normaloff>:/app/play</normaloff>:/app/play</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushRecord">
          <property name="maximumSize">
           <size>
            <width>34</width>
            <height>34</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Record selected Channel</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../common.qrc">
            <normaloff>:/app/record</normaloff>:/app/record</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushStop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>34</width>
            <height>34</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Stop Player</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../common.qrc">
            <normaloff>:/app/stop</normaloff>:/app/stop</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushBwd">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>34</width>
            <height>34</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Jump Backward</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../common.qrc">
            <normaloff>:/app/bwd</normaloff>:/app/bwd</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbxTimeJumpVal">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Time Jump Value in minutes</string>
          </property>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushFwd">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>34</width>
            <height>34</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Jump Forward</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../common.qrc">
            <normaloff>:/app/fwd</normaloff>:/app/fwd</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labSound">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../common.qrc">:/player/sound_on</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="volSlider">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
   border: 1px solid #999999;
   height: 5px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
   background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(48, 144, 22, 255), stop:0.490566 rgba(255, 242, 53, 255), stop:1 rgba(255, 0, 0, 255));
   margin: 2px 0;
}

 QSlider::handle:horizontal {
     background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #eee, 	stop:0.7 grey, stop:1 #dedede);
     border: 1px solid #5c5c5c;
     width: 4px;
     margin: -5px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
     border-radius: 2px;
 }
</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>0</number>
          </property>
          <property name="sliderPosition">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnFullScreen">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Toggle Fullscreen</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../common.qrc">
            <normaloff>:/player/leave-fullscreen</normaloff>:/player/leave-fullscreen</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QClickAndGoSlider</class>
   <extends>QSlider</extends>
   <header>qclickandgoslider.h</header>
  </customwidget>
  <customwidget>
   <class>QTimeLabel</class>
   <extends>QLabel</extends>
   <header>qtimelabel.h</header>
  </customwidget>
  <customwidget>
   <class>QMoveHandle</class>
   <extends>QLabel</extends>
   <header>qmovehandle.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../common.qrc"/>
 </resources>
 <connections/>
</ui>
